src/ui/common/src/components/workflows/ReactFlowCanvas.tsx

```diff
@@ -22,16 +22,32 @@ const ReactFlowCanvas: React.FC<ReactFlowCanvasProps> = ({
   onPaneClicked,
   switchSideSheet,
 }) => {
-  const openSideSheetState = useSelector(
-    (state: RootState) => state.openSideSheetReducer
-  );
   const dagPositionState = useSelector(
     (state: RootState) => state.workflowReducer.selectedDagPosition
   );
 
   const artifactResults = useSelector(
     (state: RootState) => state.workflowReducer.artifactResults
   );
+  
+  const currentNode = useSelector(
+    (state: RootState) => state.nodeSelectionReducer.selected
+  );
+
+  const { fitView } = useReactFlow();
+  useEffect(() => {
+    console.log(dagPositionState);
```

Comment:
remove log

do we still need this fit view?
I was hoping that by not expanding / contracting the dag we didn't have to incur these renders. 

does fit view just make things smaller now that the dag is less wide?

Reply:
Yeah, we do. Otherwise, it centers on the first couple nodes but doesn't actually center the DAG.